xsd/netex-nl-data.xsd

```diff
@@ -653,6 +653,7 @@ Bij een Version is attribuut 'version' verplicht. De waarde is gelijk aan attrib
 			<xsd:element name="Description" type="MultilingualString" minOccurs="0"/>
 			<xsd:element name="TransportMode" type="TransportmodeEnum"/>
 			<xsd:element name="TransportSubmode" type="PtSubmodeChoice" minOccurs="0"/>
+			<xsd:element name="Url" type="websiteURI" minOccurs="0"/>
```

Comment:
Which information do we expect in the Line "Url"?
The timetable? A geographical description of the route and the stops? Information over actual disturbances, incl. advice?

Reply:
Any page with more information about it.
https://www.htm.nl/dienstregeling/tram-3
https://reisinfo.gvb.nl/nl/lijnen/51